documentation/src/docs/wire-protocol.md

```diff
@@ -102,3 +139,59 @@ After receiving the "Append Setup" reply, it performs the following:
+
+## Segment Attribute API
+
+The following Paramters will be used by all the below mentioned APIs.
+
+- `RequestId`: Long (8 bytes). This field contains the client-generated _ID_ that has been propagated to identify a client request.
+- `SegmentName`: String (2 bytes) followed by that many bytes of Java's Modified UTF-8. This indicates the name of the Stream Segment.
+- `attributeId`: UUID (16 bytes).It identifies the requested attributeId.  
+
+
+| **API**      | **Description**     |
+|-------------|----------|
+|`GetSegmentAttribute`| The requested list of segment attributes.|
+|`UpdateSegmentAttribute`|  - `newValue`: Long (8 bytes). It represents the new value to be updated.|
+| |                         - `expectedValue`: Long (8 bytes). It represents |
+
+## Table Segment API
+
+The following Paramters will be used by all the below mentioned APIs.
+
+- `RequestId`: Long (8 bytes). This field contains the client-generated _ID_ that has been propagated to identify a client request.
+- `SegmentName`: String (2 bytes) followed by that many bytes of Java's Modified UTF-8. This indicates the name of the Stream Segment.
+- `delegationToken`: String (2 byte) followed by that many bytes of Java's Modified UTF-8. This was added to perform _auth_. It is an opaque-to-the-client token provided by the Controller that says it's allowed to make this call.
+
+
+| **API**      | **Description**     |
+|-------------|----------|
+|`CreateTableSegment`| To create Table segment.|
+|`MergeTableSegment`| - `Target`: String (2 bytes) followed by that many bytes of Java's Modified UTF-8. This indicates the target of the Stream Segment to be Merged.|
+|                    |- `Source`: String (2 bytes) followed by that many bytes of Java's Modified UTF-8. This indicates the source Stream Segments that needs to be Merged.|
+|`MergeSegment`|Perfomrs merging of Segments.|
+|`SealSegment`| Perfoms sealing of Segments.|
+|`SealTableSegment`| Perfoms sealing of Table Segments.|
+|`TruncateSegment`| `TruncationOffset`: Long (8 bytes). This contains the Offset for the segment to be Truncated.|
+|`DeleteSegment`| Perfoms deletion of created segments.|
+|`DeleteTableSegment`|`mustBeEmpty` (Boolean): If true, the Table Segment will only be deleted if it is empty (contains no keys).|
+
+## Table API
+
+The following Paramters will be used by all the below mentioned APIs.
+
+- `RequestId`: Long (8 bytes). This field contains the client-generated _ID_ that has been propagated to identify a client request.
+- `SegmentName`: String (2 bytes) followed by that many bytes of Java's Modified UTF-8. This indicates the name of the Stream Segment.
+- `delegationToken`: String (2 byte) followed by that many bytes of Java's Modified UTF-8. This was added to perform _auth_. It is an opaque-to-the-client token provided by the Controller that says it's allowed to make this call.
+-`TableEntries` : tableEntries;
```

Comment:
This isn't really going to help anyone know how to encode/decode this.

Reply:
@tkaitchuck OK! removed this section and similar content.